website/docs/r/api_management_api.html.markdown

```diff
@@ -82,6 +82,8 @@ The following arguments are supported:
 
 * `subscription_required` - (Optional) Should this API require a subscription key?
 
+* `type` - (Optional) The type of the API: "http", "graphql", "soap", "websocket"
```

Comment:
should this be `api_type` to make sure its exceptionally clear what "type" we are referring to?